src/Filecoin/Transaction.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright © 2017-2020 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Transaction.h"
+
+using namespace TW;
+using namespace TW::Filecoin;
+
+// encodeVaruint encodes a 256-bit number into a big endian encoding, omitting leading zeros.
+static Data encodeVaruint(const uint256_t& value) {
+    Data data;
+    encode256BE(data, value, 256);
+    size_t i = 0;
+    for (i = 0; i < data.size(); ++i)
+        if (data[i] != 0)
+            break;
```

Comment:
Please use brackets even in one-statement cases (less likely to get unexpected results when adding a statement later)